src/Platforms/tests/OMPTarget/test_ompBLAS.cpp

```diff
@@ -22,6 +22,98 @@
 
 namespace qmcplusplus
 {
+template<typename T>
+void test_gemm(const int M, const int N, const int K, const char transa, const char transb)
+{
+  const int a0 = transa == 'T' ? K : M;
+  const int a1 = transa == 'T' ? M : K;
+
+  const int b0 = transb == 'T' ? N : K;
+  const int b1 = transb == 'T' ? K : N;
```

Comment:
I feel `a0` and `a1`, `b0` and `b1` values needs to be swapped.

Reply:
I was trying to be consistent with the comment from [`test_gemv`](https://github.com/QMCPACK/qmcpack/blob/e95609ab8cf645c1397b442ca6e84cad9bc0dc52/src/Platforms/tests/OMPTarget/test_ompBLAS.cpp#L61-L63), which I interpret as describing a row-major matrix (B) passed to a function (gemv) that assumes the input is column-major.
>  // in Fortran, B[M][N] is viewed as B^T
  // when trans == 'T', the actual calculation is B * A[N] = C[M]
  // when trans == 'N', the actual calculation is B^T * A[M] = C[N]

So for the comments in `test_gemm`, I intended to label everything as if the matrices are assumed to be in row-major layout, even though the gemm function here treats them as column-major.

If the labeling is row-major, then `A[K,M] * B[N,K] = C[M,N]` is really `A^T . B^T = C`, even though the transA/transB args are both `N`.